subsys/net/ip/net_private.h

```diff
@@ -174,10 +174,12 @@ extern uint16_t net_calc_chksum_ipv4(struct net_pkt *pkt);
 #endif /* CONFIG_NET_IPV4 */
 
 #if defined(CONFIG_NET_IPV4_IGMP)
+void net_ipv4_igmp_init(struct net_if *iface);
 uint16_t net_calc_chksum_igmp(uint8_t *data, size_t len);
 enum net_verdict net_ipv4_igmp_input(struct net_pkt *pkt,
 				     struct net_ipv4_hdr *ip_hdr);
 #else
+#define net_ipv4_igmp_init(iface)
```

Comment:
We do not seem to need this define

Reply:
Thanks, I've incorporated this into my latest change.